src/components/SidebarTabs/TransferOwnership.vue

```diff
@@ -0,0 +1,176 @@
+<!--
+  - @copyright Copyright (c) 2022 Hamza Mahjoubi <hamzamahjoubi221@gmail.com>
+  -
+  - @author Hamza Mahjoubi <hamzamahjoubi221@gmail.com>
+  -
+  - @license AGPL-3.0-or-later
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<div>
+		<NcButton @click="onChangeOwner">
+			{{ t('forms', 'Transfer ownership') }}
+		</NcButton>
+		<NcModal v-if="modal"
+			ref="modal"
+			size="normal"
+			:title="'Transfer ' + form.title"
+			name="NcModal"
+			:out-transition="true"
+			@close="closeModal">
+			<div class="modal__content">
+				<h1 class="modal_section">
+					{{ t('forms', 'Transfer ') }} {{ form.title }}
+				</h1>
+				<div class="modal_section">
+					<p class="modal_text">
+						{{ t('forms', 'Account to transfer to') }}
+					</p>
+					<SharingSearchDiv :is-ownership-transfer="true" @add-share="setNewOwner" />
+					<div v-if="transferData.displayName.length > 0" class="selected_user">
+						<p>{{ transferData.displayName }}</p>
+						<NcButton type="tertiary-no-background" @click="clearSelected">
+							X
+						</NcButton>
+					</div>
+				</div>
+				<div class="modal_section">
+					<p class="modal_text">
+						{{ t('forms', 'Type ') }} {{ confirmationString }} {{ t('forms', ' to confirm') }}
```

Comment:
```suggestion
						{{ t('forms', 'Type ') }} <strong>{{ confirmationString }}</strong> {{ t('forms', ' to confirm') }}
```
I think we should highlight the text that the user has to type more.

Reply:
And if possible, make it able to copy it. 😋